test/from-draft3/to-draft4/type-has-objects-in-its-array.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "title": "`type` is present and contains objects in it",
+        "from": {
+            "type": [ {}, { "type": "string" } ],
+            "anyOf": [ { "type": [ "string", "object" ] } ]
+        },
+        "to": {
+            "type": [],
```

Comment:
Can we get rid of the empty `type`?

Reply:
Yeah, I should have another rule to cleanup empty arrays.. everywhere....